### ISSUE TITLE
Ensure transition from preview back to draft status operates correctly

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/experimenter#4495]
From a comment on https://github.com/mozilla/experimenter/pull/4482:

{quote}One thing I realized is in theory you can 'Launch to Preview' and 'Go back to draft' back and forth instantly and endlessly so we'll have to make sure we account for that in the RS sync mechanism so we don't accidentally get the backend stuck in some weird state{quote}

Not sure if there's specific work to be done here, but filing this follow up issue to capture the concern.

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/EXP-883)
┆Issue Number: EXP-883

**Comment by data-sync-user** [comment]
➤ Les Orchard commented:

Is this also blocked by EXP-887 - or can this be verified / fixed independently from GQL calls? (Kind of assuming the latter)

**Comment by data-sync-user** [comment]
➤ Jared Kerim commented:

I think we can get ahead of this when we implement the celery tasks so we don't need to track this as a separate ticket.

**Comment by lmorchard** [comment]
@jaredlockhart Can you update whichever appropriate issue with details on what needs doing here? Mostly filed this to capture the concern, but wasn't sure what exactly could go wrong that might "accidentally get the backend stuck in some weird state"

**Comment by data-sync-user** [comment]
➤ Les Orchard commented:

Re-opening this: Turns out there's a bit more to do here as transition back to Draft from Preview doesn't quite work without a few more tweaks.

https://github.com/mozilla/experimenter/commit/155e4950b76d52af854cc848cb01697cc2a95dac#diff-da5192d8fdd27a14107b117c1f4fbfbbc6cbee34c61bc19dd273fc612781f7ebR389-R397

https://github.com/mozilla/experimenter/commit/155e4950b76d52af854cc848cb01697cc2a95dac#diff-17cceb97941fb3f54f365e750e7d64e96d53ee427a7bc952e328bdb8be96a9daR311
